packages/payment-integration-api/src/util-types/index.ts

```diff
@@ -1,3 +1,4 @@
-export { default as Omit } from "./omit";
-export { default as RequestOptions } from "./request-options";
-export { default as RequireAtLeastOne } from "./require-at-least-one";
+export { default as isApplePayWindow, ApplePayWindow } from './is-apple-pay-window'
```

Comment:
`isApplePayWindow` and `ApplePayWindow` should belong to the `apple-pay-integration` package instead. Also, I just noticed that `PaymentMethodActionCreator` in `core` references `isApplePayWindow`. That ideally should be refactored as `core` shouldn't be aware of individual payment methods.

Reply:
Ideally it should be in `apple-pay-integration`, but since we filter payment methods on whether they are supported or not hence we reference it there. Based on that I decided to move this to a shared type so can be used. 

We can look into the uses, and have another look to see if we can remove that later.